lib/index.js

```diff
@@ -47,6 +47,10 @@ const exceptionWrapper = function tryCatchWrapper(funcToWrap) {
       process.exit(0);
     } catch (err) {
       logger.error(err.message);
+      if (process.env.BINARIS_LOG_LEVEL === 'info' || !process.env.BINARIS_LOG_LEVEL) {
+        // eslint-disable-next-line max-len
+        logger.warn('\nTip: You can export BINARIS_LOG_LEVEL=[silly|debug|verbose|info|warn|error] to view debug logs');
```

Comment:
Is it really true e.g. for `error`?

Reply:
I mean those are the logging levels we have defined. Of course error level works.

```
const colorLevels = {
  silly: 'white',
  debug: 'white',
  verbose: 'white',
  info: 'white',
  warn: 'yellow',
  error: 'red',
};
```

Reply:
Again, the code is easy to understand (yay!).  But I disagree with what it does.  The context in which I see the hint message is that I just had an error message.  And this is when you tell me I could "export `BINARIS_LOG_LEVEL=...error... to view debug logs`".  It's a good time to suggest `verbose` or `debug` or `silly` (even then, _only_ those that will actually give extra information!), but it's hardly a good time to tell me about levels that are more restrictive than I already have.

Reply:
Understood.